nixos/modules/programs/environment.nix

```diff
@@ -45,7 +45,7 @@ in
         GTK_PATH = [ "/lib/gtk-2.0" "/lib/gtk-3.0" "/lib/gtk-4.0" ];
         XDG_CONFIG_DIRS = [ "/etc/xdg" ];
         XDG_DATA_DIRS = [ "/share" ];
-        LIBEXEC_PATH = [ "/lib/libexec" ];
```

Comment:
If this does not exist, and hasn't been causing issues, can we just drop it?

Reply:
It is needed for some Plasma stuff to work properly, notably drkonqi.

Reply:
But I mean it has been pointing to the wrong path, and plasma was happy.

Reply:
Yep because we never had drkonqi. And now I want to have it for 6, but also in general.